docs/proposals/excluding-drives.md

```diff
@@ -0,0 +1,105 @@
+# Proposal: Specific drives excluding
+
+Last updated: 07.02.2022
+
+
+## Abstract
+
+CSI Baremetal needs to have API for excluding specific drives from the common storage pool. 
+Excluded drives won't be used for allocating new Volumes or LVGs.
+
+## Background
+
+User stories:
+1. Deploy another CSI driver in kube-cluster.
+2. Reserve space for increasing root partition.
+3. A kube-node is used for 2 or more kube-clusters (or not only for cluster).
```

Comment:
Is that possible at all?

Reply:
Yes, but I think it is not a widely used scenario. Do you have other ideas why we need this feature?

Reply:
I think that main use case that customer don't want to use some disks for whatever reason:
1. for another application not managed by CSI
2. as a workaround for any disk detection issues in CSI

Reply:
fixed